__tests__/cmds/logout.test.ts

```diff
@@ -6,6 +6,10 @@ import configStore from '../../src/lib/configstore';
 const cmd = new Command();
 
 describe('rdme logout', () => {
+  afterEach(() => {
+    configStore.clear();
```

Comment:
Oh is us not clearing this out why sometimes the login test flakes out?

Reply:
I think so 😬  the test bed seems a little flakier now that I added so many tests messing with `configstore` so I'm hoping this addresses that